swift/ql/lib/codeql/swift/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -125,6 +147,8 @@ private module Cached {
     or
     // flow through `!`
     nodeFrom.asExpr() = nodeTo.asExpr().(ForceValueExpr).getSubExpr()
+    or
+    FlowSummaryImpl::Private::Steps::summaryLocalStep(nodeFrom, nodeTo, true)
```

Comment:
```suggestion
    // flow through a flow summary (extension of `SummaryModelCsv`)
    FlowSummaryImpl::Private::Steps::summaryLocalStep(nodeFrom, nodeTo, true)
```
I take it there's no QL equivalent to `SummaryModelCsv` if we need more power (as an extension of `RemoteFlowSource` is for `SourceModelCsv`)?  We can always add more complex cases to `defaultAdditionalTaintStep` of course, like we previously did for `URL.init`.